pkg/informer/informer.go

```diff
@@ -532,25 +552,52 @@ func (d *DynamicDiscoverySharedInformerFactory) updateInformers() {
 			klog.V(4).InfoS("Unable to notify discovery subscriber - channel full", "id", id)
 		}
 	}
-
 }
 
 // gvrsToDiscoveryData returns "fake"/simulated discovery data for all the resources covered by the factory. It only
 // includes enough data in each APIResource to support what kcp currently needs (scheduling, placement, quota).
-func gvrsToDiscoveryData(gvrs map[schema.GroupVersionResource]struct{}) []*metav1.APIResourceList {
+func (d *DynamicDiscoverySharedInformerFactory) gvrsToDiscoveryData(gvrs map[schema.GroupVersionResource]struct{}) []*metav1.APIResourceList {
 	var discoveryData []*metav1.APIResourceList
 	gvResources := make(map[schema.GroupVersion][]metav1.APIResource)
+	builtInTypes := builtInInformableTypes()
```

Comment:
We calculate the `gvrs` parameter from this before passing it into `gvrsToDiscoveryData` - why do we end up calling this again inside?

Reply:
Done 👍🏼.